### PR TITLE
Adds Expand/Collapse Preview option

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2379](https://openedx.atlassian.net/browse/DOC-2379)

Re-adds previously reviewed text and images for the video preview option. Sanity check please!
### Date Needed

By 1 pm today
### Reviewers
- [x] Subject matter expert: @dsjen 
- [X] Subject matter expert: @dan-f 
- [x] Doc team review (sanity check): @catong or @srpearce 
### Testing
- [x] Ran make html without warnings or errors
### Post-review
- [X] Add description to release notes task as a comment
- [x] Update change log
- [x] Squash commits

### CODE REVIEW

**Comment by srpearce** [comment]
On it...

**Comment by srpearce** [comment]
:+1:

**Comment by dsjen** [comment]
LGTM!  :movie_camera:   I restarted your build because it failed--seemed like a network issue.

File: docs/en_us/dashboard/source/engagement/Engagement_Video.rst [hunk 1]
```diff
@@ -139,27 +139,27 @@ happens at the 40 second mark.
  :alt: A chart showing a noticeable increase in the number of replays 40
   seconds into the video.
 
-To find out what that segment of the video contains, you select **View Live**
```
Comment (dan-f) [inline docs/en_us/dashboard/source/engagement/Engagement_Video.rst:142]: I just noticed this. The "View Live" option is still available. Should we still document it?

**Comment by dan-f** [comment]
:+1: other than my above question.

File: docs/en_us/dashboard/source/engagement/Engagement_Video.rst (hunk 1 above)
Comment (lamagnifica) [inline docs/en_us/dashboard/source/engagement/Engagement_Video.rst:142]: ah, @dan-f , we do!
see line 242. (in this particular use case, using the preview right in Insights make sense. We have another scenario where it makes more sense to backtrack to the LMS and then back to Studio)

**Comment by dsjen** [comment]
@lamagnifica https://github.com/edx/edx-analytics-dashboard/pull/384 fixes the build error.  Once it's merged, please rebase on it.

**Comment by dsjen** [comment]
Fix is merged!  Please rebase.

File: docs/en_us/dashboard/source/engagement/Engagement_Video.rst (hunk 1 above)
Comment (dan-f) [inline docs/en_us/dashboard/source/engagement/Engagement_Video.rst:142]: Thanks for clarifying. That makes a lot of sense.

**Pull request merged**